### PR TITLE
Versioning

### PR DESCRIPTION
**Pull request by joejstuart**

### CODE REVIEW

**Comment by joejstuart** [comment]
- Only push image to docker hub when merged with master.
- Add a file called VERSION to each container. If the version in the file is bumped, push an image when merged to master tagged with the contents of VERSION along with tagged "latest".
- If the VERSION file isn't update and a PR is merged to master push an image tagged only with "latest".

@arilivigni @dirgim Please take a look and let me know what you think of this workflow. Thanks

File: ansible/VERSION [hunk 1]
```diff
@@ -0,0 +1 @@
+0.0.1
```
Comment (arilivigni) [inline ansible/VERSION]: Will the versions be changed based on what is the latest on dockerhub

File: linchpin/VERSION [hunk 2]
```diff
@@ -0,0 +1 @@
+0.0.1
```
Comment (arilivigni) [inline linchpin/VERSION]: Will the versions be changed based on what is the latest on dockerhub

File: ansible/VERSION (hunk 1 above)
Comment (joejstuart) [reply ansible/VERSION]: In this change you would have to update that file manually to trigger a new version. The only automated process would be the "latest" tag being added on a merge to master. I wasn't sure how to do automated semantic versioning.
My other thought is to let the container versions follow the release schedule of the shared libraries. That way we could say everything on version x has been tested together and in between releases update the patch version if necessary. In the latter the patch version would be updated based on dockerhub.

Comment (arilivigni) [reply ansible/VERSION]: I think that makes sense to get some consistency with the libraries and infra but may be very difficult to maintain. My initial question was around more how does this reflect in regards to the versioning we already have out on dockerhub for ansible-executor and linchpin-executor? Does this reset the version?

Comment (joejstuart) [reply ansible/VERSION]: I've bumped the version numbers so they won't conflict with the existing image versions.

**Pull request merged**